test/built-ins/Temporal/Duration/compare/precision-exact-mathematical-values-1.js

```diff
@@ -0,0 +1,68 @@
+// Copyright (C) 2022 André Bargull. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-temporal.duration.compare
+description: >
+  Duration components are precise mathematical integers.
+info: |
+  Temporal.Duration.compare ( one, two [ , options ] )
+
+  ...
+  7. If any of one.[[Years]], two.[[Years]], one.[[Months]], two.[[Months]], one.[[Weeks]], or
+     two.[[Weeks]] are not 0, then
+    a. Let unbalanceResult1 be ? UnbalanceDurationRelative(one.[[Years]], one.[[Months]],
+       one.[[Weeks]], one.[[Days]], "day", relativeTo).
+  ...
+  9. Let ns1 be ! TotalDurationNanoseconds(days1, one.[[Hours]], one.[[Minutes]], one.[[Seconds]],
+     one.[[Milliseconds]], one.[[Microseconds]], one.[[Nanoseconds]], shift1).
+ 10. Let ns2 be ! TotalDurationNanoseconds(days2, two.[[Hours]], two.[[Minutes]], two.[[Seconds]],
+     two.[[Milliseconds]], two.[[Microseconds]], two.[[Nanoseconds]], shift2).
+ 11. If ns1 > ns2, return 1𝔽.
+ 12. If ns1 < ns2, return -1𝔽.
+ 13. Return +0𝔽.
```

Comment:
```suggestion
  10. Let ns2 be ! TotalDurationNanoseconds(days2, two.[[Hours]], two.[[Minutes]], two.[[Seconds]],
     two.[[Milliseconds]], two.[[Microseconds]], two.[[Nanoseconds]], shift2).
  11. If ns1 > ns2, return 1𝔽.
  12. If ns1 < ns2, return -1𝔽.
  13. Return +0𝔽.
```
(dedenting by one position seems to mess up the YAML parser)